src/api/client.js

```diff
@@ -1,5 +1,6 @@
 /* eslint import/prefer-default-export: "off" */
 import Cookies from 'js-cookie';
+import 'whatwg-fetch';  // fetch polyfill
```

Comment:
Same as what's done in marketing, https://github.com/edx/edx-mktg/blob/master/docroot/sites/all/themes/atedx/scripts/react/data/actions/autoSuggest.jsx#L1